pkg/test/env/config.default.yml

```diff
@@ -0,0 +1,21 @@
+test:
+  defaults:
+    images:
+      ddb:
+        repository: "amazon/dynamodb-local"
+        tag: "1.21.0"
+      localstack:
+        repository: "localstack/localstack"
+        tag: "1.4"
+      mysql:
+        repository: "mysql/mysql-server"
+        tag: "8.0"
+      redis:
+        repository: "redis"
+        tag: "7-alpine"
+      s3:
+        repository: "minio/minio"
+        tag: "RELEASE.2023-03-09T23-16-13Z"
+      wiremock:
+        repository: "wiremock/wiremock"
+        tag: "3.3.1-alpine"
```

Comment:
please change to non-alpine to ensure arm64 compatibility

Reply:
changed to `3.4.1`